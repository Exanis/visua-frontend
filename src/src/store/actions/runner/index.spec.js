import * as Actions from './index';

describe('Runner Actions', () => {
    it('getRunnerList', () => {
        expect(Actions.getRunnerList(1, 'test')).toMatchSnapshot();
    });
    it('onRunnerList', () => {
        expect(Actions.onRunnerList(['first', 'second'])).toMatchSnapshot();
    });
});
