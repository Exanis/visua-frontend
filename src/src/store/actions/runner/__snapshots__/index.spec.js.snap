// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Runner Actions getRunnerList 1`] = `
Object {
  "page": 1,
  "search": "test",
  "type": "runner.list",
}
`;

exports[`Runner Actions onRunnerList 1`] = `
Object {
  "runner": Array [
    "first",
    "second",
  ],
  "type": "runner.onList",
}
`;
